cijoe/tests/logic/test_xnvme_tests_enum.py

```diff
@@ -7,6 +7,8 @@
 def test_open(cijoe, device, be_opts, cli_args):
     if be_opts["be"] == "ramdisk":
         pytest.skip(reason="[be=ramdisk] does not support enumeration")
+    if be_opts["be"] == "driverkit":
+        pytest.skip(reason="[be=ramdisk] does not support repeatedly enumerating")
```

Comment:
The skip reason here needs to be corrected `[be=driverkit]` instead of `[be=ramdisk]`